release-notes/6.0/api-diff/.Net/6.0.0_System.md

```diff
@@ -0,0 +1,336 @@
+# System
+
+``` diff
+ namespace System {
+     public class ArgumentNullException : ArgumentException {
++        public static void ThrowIfNull(object argument, string paramName = null);
+     }
+     public abstract class Array : ICloneable, ICollection, IEnumerable, IList, IStructuralComparable, IStructuralEquatable {
++        public static int MaxLength { get; }
++        public static void Clear(Array array);
+     }
+     public abstract class Attribute {
+-        public static Attribute[] GetCustomAttributes(MemberInfo element, Type type);
++        public static Attribute[] GetCustomAttributes(MemberInfo element, Type attributeType);
+-        public static Attribute[] GetCustomAttributes(MemberInfo element, Type type, bool inherit);
++        public static Attribute[] GetCustomAttributes(MemberInfo element, Type attributeType, bool inherit);
+     }
+     public static class BitConverter {
++        public static ulong DoubleToUInt64Bits(double value);
++        public static byte[] GetBytes(Half value);
++        public static short HalfToInt16Bits(Half value);
++        public static ushort HalfToUInt16Bits(Half value);
++        public static Half Int16BitsToHalf(short value);
++        public static uint SingleToUInt32Bits(float value);
++        public static Half ToHalf(byte[] value, int startIndex);
++        public static Half ToHalf(ReadOnlySpan<byte> value);
++        public static bool TryWriteBytes(Span<byte> destination, Half value);
++        public static Half UInt16BitsToHalf(ushort value);
++        public static float UInt32BitsToSingle(uint value);
++        public static double UInt64BitsToDouble(ulong value);
+     }
+-    public readonly struct Byte : IComparable, IComparable<byte>, IConvertible, IEquatable<byte>, IFormattable
++    public readonly struct Byte : IComparable, IComparable<byte>, IConvertible, IEquatable<byte>, IFormattable, ISpanFormattable
+-    public readonly struct Char : IComparable, IComparable<char>, IConvertible, IEquatable<char> {
++    public readonly struct Char : IComparable, IComparable<char>, IConvertible, IEquatable<char>, IFormattable, ISpanFormattable {
++        public static bool IsAscii(Char c);
```

Comment:
Capital `Char` rather than the language keyword `char`?

Reply:
I agree I think it should be ``char``.